infrastructure-playbooks/purge-cluster.yml

```diff
@@ -48,10 +49,23 @@
     - "{{ rgw_group_name }}"
     - "{{ rbdmirror_group_name }}"
     - "{{ nfs_group_name }}"
+    - "{{ client_group_name }}"
 
   become: true
 
   tasks:
+  - name: test if ceph-detect-init command exist
+    command: command -v ceph-detect-init
+    changed_when: false
+    failed_when: false
+    register: ceph_detect_init_command
+
+  - name: exit playbook, if ceph-detect-init command does not exist
+    debug:
+      msg: "The ceph-detect-init command is not available, please install ceph-base."
```

Comment:
Well only if the playbook is ran multiple times then yes... It will also fail to run the command. So I guess we have the same problem in both cases.